starkinfra/dynamicbrcode/__dynamicbrcode.py

```diff
@@ -0,0 +1,112 @@
+from starkcore.utils.resource import Resource
+from starkcore.utils.checks import check_datetime, check_date
+from ..utils import rest
+
+
+class DynamicBrcode(Resource):
+    """# DynamicBrcode object
+    A DynamicBrcode stores account information in the form of a Pix Key and can easily be used to create Pix transactions.
+    When you initialize a DynamicBrcode, the entity will not be automatically
+    created in the Stark Infra API. The 'create' function sends the objects
+    to the Stark Infra API and returns the created object.
+    ## Parameters (required):
+    - name [string]: receiver's name. ex: "Tony Stark"
+    - city [string]: receiver's city name. ex: "Rio de Janeiro"
+    - external_id [string]: string that must be unique among all your dynamic brcodes. Duplicated external IDs will cause failures. ex: "my-internal-id-123456"
+    ## Parameters (optional):
+    - type [string]: type of the DynamicBrcode. ex: "instant", "due"
```

Comment:
- type [string]: type of the DynamicBrcode. Options: "instant" or "due", Default: "instant"